fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/api/FedoraToOcflObjectIndex.java

```diff
@@ -36,34 +38,52 @@
      * Contrast this  with an Archival Group example:  if you pass in "my/archival-group/binary/fcr:metadata" the
      * resource returned in the mapping would be "my/archival-group".
      *
+     * @param sessionId id of the current session, or null for read-only.
      * @param fedoraResourceIdentifier the fedora resource identifier
      * @return the mapping
      * @throws FedoraOCFLMappingNotFoundException when no mapping exists for the specified identifier.
      */
-    FedoraOCFLMapping getMapping(final String fedoraResourceIdentifier) throws FedoraOCFLMappingNotFoundException;
+    FedoraOCFLMapping getMapping(final String sessionId, final String fedoraResourceIdentifier) throws
```

Comment:
While I don't disagree, the persistence codebase consistent refers to it as a "session". Perhaps we should have a follow up jira to unify the language?

Reply:
I am fine with a follow-up JIRA for name unification. Currently, it seems like we are open to potentially challenging to track-down bugs since we inter-mix sessions IDs and transaction IDs... is there a difference?

Reply:
No, there is no difference

Reply:
https://jira.lyrasis.org/browse/FCREPO-3388

Reply:
New JIRA: https://jira.lyrasis.org/browse/FCREPO-3388